ebu_tt_live/documents/test/converter_ericsson1.xml

```diff
@@ -12,7 +12,7 @@
                 <ebuttm:documentOriginalProgrammeTitle>tgv</ebuttm:documentOriginalProgrammeTitle>
                 <ebuttm:documentOriginalEpisodeTitle>tgv</ebuttm:documentOriginalEpisodeTitle>
                 <ebuttm:documentCreationDate>2016-09-06</ebuttm:documentCreationDate>
-                <ebuttm:documentRevisionNumber/>
+                <!--<ebuttm:documentRevisionNumber/>-->
```

Comment:
Don't like the artefact.

Reply:
Yeah, it's not nice but since it was an Ericsson generated example it felt bad to remove it completely, which would have hidden the fact that their tool generated it. On balance I preferred to keep the artefact then remove it altogether. I'm going to add a note into the comment explaining this better.